packages/encryption/tests/encryption.test.ts

```diff
@@ -645,3 +661,42 @@ test('Sign msg using @noble/secp256k1 and verify signature using elliptic/secp25
   expect(ellipticVerifyResult).toBeTruthy();
 });
 
+test('Verify compatibility @scure/bip39 <=> bitcoinjs/bip39', () => {
+  // Consider an entropy
+  const entropy = '00000000000000000000000000000000';
+  // Consider same entropy in array format
+  const entropyUint8Array = new Uint8Array(entropy.split('').map(Number));
+
+  // entropyToMnemonicBip39 imported from bitcoinjs/bip39
+  const bip39Mnemonic = entropyToMnemonicBip39(entropy);
+  // entropyToMnemonic imported from @scure/bip39
+  const mnemonic = entropyToMnemonic(entropyUint8Array, wordlist);
+
+  //Phase 1: Cross verify mnemonic validity: @scure/bip39 <=> bitcoinjs/bip39
+
+  // validateMnemonic imported from @scure/bip39
+  expect(validateMnemonic(bip39Mnemonic, wordlist)).toEqual(true);
+  // validateMnemonicBip39 imported from bitcoinjs/bip39
+  expect(validateMnemonicBip39(mnemonic)).toEqual(true);
+
+  // validateMnemonic imported from @scure/bip39
+  expect(validateMnemonic(mnemonic, wordlist)).toEqual(true);
+  // validateMnemonicBip39 imported from bitcoinjs/bip39
+  expect(validateMnemonicBip39(bip39Mnemonic)).toEqual(true);
+
+  //Phase 2: Get back entropy from mnemonic and verify @scure/bip39 <=> bitcoinjs/bip39
+
+  // mnemonicToEntropy imported from @scure/bip39
+  expect(mnemonicToEntropy(mnemonic, wordlist)).toEqual(entropyUint8Array);
+  // mnemonicToEntropyBip39 imported from bitcoinjs/bip39
+  expect(mnemonicToEntropyBip39(bip39Mnemonic)).toEqual(entropy);
+  // mnemonicToEntropy imported from @scure/bip39
+  expect(Buffer.from(mnemonicToEntropy(bip39Mnemonic, wordlist)).toString('hex')).toEqual(entropy);
+  // mnemonicToEntropyBip39 imported from bitcoinjs/bip39
+  const entropy64Byte = mnemonicToEntropyBip39(mnemonic);
+  // Convert entropy to 32 byte due to 24 words mnemonic
+  const entropy32Byte = new Uint8Array(entropy64Byte.split('').filter((_, i) => i % 2 !== 0).map(Number))
+  // entropy should match with entropyUint8Array
+  expect(entropy32Byte).toEqual(entropyUint8Array);
+});
+
```

Comment:
Can these tests be made against test vectors, rather than the bip39 library? That way, we won't need to include bip39 as a devDependency.

Reply:
✅

Reply:
Is this resolved?

Reply:
Yes. Done!